Shared/Samples/Create buffers around points/CreateBuffersAroundPointsView.swift

```diff
@@ -57,29 +69,29 @@ struct CreateBuffersAroundPointsView: View {
                     .disabled(model.bufferPoints.isEmpty)
                 }
             }
-            .alert("Buffer Radius", isPresented: $model.inputBoxIsPresented, actions: {
-                TextField("100", text: $model.radiusInput)
+            .alert("Buffer Radius", isPresented: $inputBoxIsPresented, actions: {
+                TextField("100", text: $radiusInput)
```

Comment:
Manually converting to/from string and double is tedious and error prone. Thankfully, SwiftUI has our back. Instead of storing the user input, store the buffer radius:

```swift
@State private var bufferRadius: Double = 100
```

And then pass that when creating the `Text` instance:

```swift
TextField("radius in miles", value: $bufferRadius, format: .number)
```

Reply:
> If the user stops editing the text in an invalid state, the text field updates the field’s text to the last known valid value. [doc](https://developer.apple.com/documentation/swiftui/textfield/init(value:format:prompt:label:)-99ntf)

This is an interesting behavior. If the default value is 100 and I tap the backspace 3 times, the resulting value is 1. Which might be surprising to a user who expect an invalid input. But it is not a big deal.